tests/blackbox/testlib/utils.py

```diff
@@ -107,6 +107,8 @@ def exec_command(cmd):
         print("STDERR= %s" % stderr_text)
 
     assert expected_exit_code == exit_code
+    if expected_exit_code != exit_code:
```

Comment:
For here, it makes sense to stow all the information that is in those print statements above this line in the RuntimeError, and to get rid of the print statements entirely.

Reply:
I forgot to remove the "assert" line above; fixed that, and added the print statements.

Reply:
Resolved.